### DISCUSSION TITLE
How to install extensions via cli?

### DISCUSSION THREAD

**Discussion by Jeeppler** [eclipse-theia/theia#13785] [Q&A]
How do you install extensions via cli in Eclipse Theia IDE (formerly Blueprint)?

The [install extension cli command was added in Theia 1.47.0](https://github.com/eclipse-theia/theia/pull/13421). From my understanding it should work like this:

~~~
theia --install-extension "asciidoctor.asciidoctor-vscode"
~~~

However, if I execute this in Theia 1.49.1, the IDE will start and close quickly, but the actual extension is not installed. How to use the Theia CLI in Theia Blueprint?

**Comment by msujew** [comment]
cc @tsmaeder

**Reply by tsmaeder** [reply]
@Jeeppler what do you seen in the console if you start from command line? Also: 1.47.0 is three releases out of date 🤷

**Reply by Jeeppler** [reply]
@tsmaeder I am using Theia 1.49.1, which I think is the latest release as of now.

If I run this command:

~~~
$ theia --install-extension "asciidoctor.asciidoctor-vscode"

# or this

$ theia --install-extension "rust-lang.rust"
~~~

I see nothing in the console, neither an error nor any output. The only thing that happens is, that TheiaIDE starts and immediately closes.

In addition, I do not see any output if run:

~~~
$ theia check:theia-version
~~~

or 

~~~
$ theia --help
~~~

NOTE: Not sure if `check:theia-version` or `--help` should work.

OS: GNU/Linux Ubuntu 22.04 x86-64
